git/essentials/features-iii/using-tags-for-version-control.md

```diff
@@ -13,6 +13,12 @@ type: normal
 
 category: must-know
 
+aspects:
+  
+  - deep
+  - workout
+  - obscura
```

Comment:
Would remove `obscura`, this is a pretty powerful component of git's ecosystem and necessary for using some features on sites like GitHub, for example; releases.